SSA/Core/Util.lean

```diff
@@ -91,3 +93,21 @@ elab "print_goal_as_error " : tactic => do
   let target ← Lean.Elab.Tactic.getMainTarget
   throwError target
   pure ()
+
+-- TODO: should these go into Std?
+def Vector.ofList {α : Type u} (l : List α) : Vector α l.length :=
+match l with
+  | List.nil => Vector.nil
+  | List.cons l ls => Vector.cons l (Vector.ofList ls)
```

Comment:
`Vector` is defined as a subtype of list, so (some variation of) this should work:

```suggestion
⟨l, rfl⟩
```